README.md

```diff
@@ -30,6 +30,28 @@ Note: The PoC application shown here, is ideal for demonstration purposes only.
 
 ---
 
+## E2E Testing
+
+For e2e testing we are using cypress and **docker image** - `image: "cypress/included:3.2.0"` which comes with preinstalled cypress. Cypress config and tests can be found in `./doocker/` folder. Test flows or Cypress specs are located in `./docker/cypress/integration/` folder. Each spec should represent a flow e.g.
```

Comment:
Found in `./doocker/` folder ?

Reply:
refresh the page please, it was commited like 20mins ago
<img width="783" alt="Screenshot 2022-03-30 at 14 03 41" src="https://user-images.githubusercontent.com/11794402/160840899-f6050c61-fd22-4085-b863-eaf4f39802e0.png">